activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,33 @@
+---
+type: Resource
+---
+
+# What information to collect about members for member relations 
+
+## Introduction
+
+This resource outlines the information that Foundation for Public Code staff use to map member relations.
+
+The information collected is used to structure and keep track of our ongoing relationships with members. It is based on our [mission](../../organization/mission.md) and the first iterations of the [founding member proposal template](founding-membership-proposal.md). In line with our communication principles, it is concise and hypertextual where possible. 
+
+As information collected may include personal or sensitive information, this is not public but stored in a private and secure environment.
+
+## Information collected 
+
+* Background:
+   * History: `Provide a short outline of the organizations' history with open and collaborative code, including for example notable success stories, failures, or strong political movements`
+   * Aligned projects: ```Provide a short description of notable public-purpose open and collaborative software and policy```
+   * Opportunity projects: ```Provide a short description of notable software and policy with strong potential to become more open, collaborative and/or public-purpose```
+   * Additional comments: ```Provide a short description of other initiatives not directly related to, but adjacent to our mission```
+* Relationship to the Foundation for Public Code
+   * Decision makers: ```Outline the top two or three budget holders, policy makers and managers who decide on Foundation membership```
```

Comment:
These items in this section are written from the perspective of getting an org to become a member, not from managing an existing relationship:
- "who decide on Foundation membership"
- "who can help promote our mission"
- "for whom we can test our value propositions"

Since we don't know for sure yet what we need to manage an ongoing relationship, this could be retitled "Proposal for collecting information for new member relations"

Reply:
fix to make it about 'potential' members